library/desktop/yast-settings.menu

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE Menu PUBLIC "-//freedesktop//DTD Menu 1.0//EN"
+"http://www.freedesktop.org/standards/menu-spec/1.0/menu.dtd">
+<Menu>
+	<Name>YaST</Name>
+	<Directory>X-SuSE-YaST.directory</Directory>
+	<Include>
+		<And>
+		<Category>X-SuSE-YaST</Category>
+		<Not>
+		<Category>X-SuSE-YaST-Hardware</Category>
+		<Category>X-SuSE-YaST-HighAvailability</Category>
+		<Category>X-SuSE-YaST-Misc</Category>
+		<Category>X-SuSE-YaST-Net_advanced</Category>
+		<Category>X-SuSE-YaST-Security</Category>
+		<Category>X-SuSE-YaST-Software</Category>
+		<Category>X-SuSE-YaST-Support</Category>
+		<Category>X-SuSE-YaST-System</Category>
+		<Category>X-SuSE-YaST-Virtualization</Category>
+		</Not>
+		</And>
+	</Include>
+	<Menu>
+		<Name>Software</Name>
+		<Directory>X-SuSE-YaST-Software.directory</Directory>
+		<Include>
+			<Category>X-SuSE-YaST-Software</Category>
+		</Include>
+	</Menu>
+	<Menu>
+		<Name>System</Name>
+		<Directory>X-SuSE-YaST-System.directory</Directory>
+		<Include>
+			<Category>X-SuSE-YaST-System</Category>
+		</Include>
+	</Menu>
+	<Menu>
+		<Name>Hardware</Name>
+		<Directory>X-SuSE-YaST-Hardware.directory</Directory>
+		<Include>
+			<Category>X-SuSE-YaST-Hardware</Category>
+		</Include>
+	</Menu>
+	<Menu>
+		<Name>High Availability</Name>
+		<Directory>X-SuSE-YaST-HighAvailability.directory</Directory>
+		<Include>
+			<Category>X-SuSE-YaST-High_Availability</Category>
+		</Include>
+	</Menu>
+	<Menu>
+		<Name>Network Services</Name>
+		<Directory>X-SuSE-YaST-NetworkServices.directory</Directory>
+		<Include>
+			<Category>X-SuSE-YaST-Net_advanced</Category>
+		</Include>
+	</Menu>
+	<Menu>
+		<Name>Security</Name>
+		<Directory>X-SuSE-YaST-Security.directory</Directory>
+		<Include>
+			<Category>X-SuSE-YaST-Security</Category>
+		</Include>
+	</Menu>
+	<Menu>
+		<Name>Virtualization</Name>
+		<Directory>X-SuSE-YaST-Virtualization.directory</Directory>
+		<Include>
+			<Category>X-SuSE-YaST-Virtualization</Category>
+		</Include>
+	</Menu>
+	<Menu>
+		<Name>Support</Name>
+		<Directory>X-SuSE-YaST-Support.directory</Directory>
+		<Include>
+			<Category>X-SuSE-YaST-Support</Category>
+		</Include>
+	</Menu>
+	<Menu>
+		<Name>Miscellaneous</Name>
+		<Directory>X-SuSE-YaST-Misc.directory</Directory>
+		<Include>
+			<Category>X-SuSE-YaST-Misc</Category>
+		</Include>
+	</Menu>
```

Comment:
I have question about this approach. We now have some yast modules that defines own category and we are not aware about it ( at least almost not aware as it is private yast modules, just see some bug reports ). How they can add new categories to yast menus?